deploy/sdk/src/dynamo/sdk/cli/serve_dynamo.py

```
@@ -298,6 +300,14 @@ async def system_app_worker():
         await instanceReady.wait()
         if not service.system_app:
             raise ValueError("System app not defined for service")
+        # Setup nats client for readyness and liveliness health checks.
+        try:
+            nats_client = NATS()
+            await nats_client.connect("nats://localhost:4222")
+            set_nats_client(nats_client)
+            logger.info("Connected to NATS and set global client for probes")
+        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)